tests/Oryx.BuildImage.Tests/CommandTests/OryxCommandTest.cs

```diff
@@ -193,6 +193,7 @@ public void BuildImage_CanExec_MultipleCommands_WithOlderToolVersions()
                     "'{\"engines\": {\"node\": \"" + expectedNodeVersion + "\"}}'")
                 .CreateFile($"{appPath}/{PhpConstants.ComposerFileName}",
                     "'{\"require\": {\"php\": \"" + expectedPhpVersion + "\"}}'")
+                .AddCommand("export ORYX_SDK_STORAGE_BASE_URL=https://oryxsdksdev.blob.core.windows.net")
```

Comment:
.AddEnvironmentVariables()

Reply:
it will be adddefaulttestenvironment instead .. thanks !